TM1py/Services/GitService.py

```diff
@@ -205,60 +185,65 @@ def git_pull(self, branch: str, force: bool = None, execute: bool = None, userna
 
-            # A git plan can either be a PushPlan or a PullPlan, these have slightly different variables, so we need to handle those differently
+            # A git plan can either be a PushPlan or a PullPlan, these have slightly different variables,
+            # so we need to handle those differently
             if plan.get('@odata.type') == '#ibm.tm1.api.v1.GitPushPlan':
                 plan_new_branch = plan.get('NewBranch')
                 plan_source_files = plan.get('SourceFiles')
 
                 new_commit = GitCommit(
-                            commit_id=plan.get('NewCommit').get('ID'),
-                            summary=plan.get('NewCommit').get('Summary'),
-                            author=plan.get('NewCommit').get('Author'))
+                    commit_id=plan.get('NewCommit').get('ID'),
+                    summary=plan.get('NewCommit').get('Summary'),
+                    author=plan.get('NewCommit').get('Author'))
 
                 parent_commit = GitCommit(
-                            commit_id=plan.get('ParentCommit').get('ID'),
-                            summary=plan.get('ParentCommit').get('Summary'),
-                            author=plan.get('ParentCommit').get('Author'))
+                    commit_id=plan.get('ParentCommit').get('ID'),
+                    summary=plan.get('ParentCommit').get('Summary'),
+                    author=plan.get('ParentCommit').get('Author'))
 
                 current_plan = GitPushPlan(
-                            plan_id=plan_id, branch=plan_branch, force=plan_force,
-                            new_branch=plan_new_branch, new_commit=new_commit,
-                            parent_commit=parent_commit, source_files=plan_source_files)
+                    plan_id=plan_id, branch=plan_branch, force=plan_force,
+                    new_branch=plan_new_branch, new_commit=new_commit,
+                    parent_commit=parent_commit, source_files=plan_source_files)
 
             elif plan.get('@odata.type') == '#ibm.tm1.api.v1.GitPullPlan':
 
                 plan_commit = GitCommit(
-                                commit_id=plan.get('Commit').get('ID'), 
-                                summary=plan.get('Commit').get('Summary'), 
-                                author=plan.get('Commit').get('Author'))
+                    commit_id=plan.get('Commit').get('ID'),
+                    summary=plan.get('Commit').get('Summary'),
+                    author=plan.get('Commit').get('Author'))
 
                 plan_operations = plan.get('Operations')
-                current_plan = GitPullPlan(plan_id=plan_id, branch=plan_branch, force=plan_force, commit=plan_commit, operations=plan_operations)
+                current_plan = GitPullPlan(plan_id=plan_id, branch=plan_branch, force=plan_force, commit=plan_commit,
+                                           operations=plan_operations)
 
             else:
-                raise ValueError("Invalid plan detected")
-            
```

Comment:
Can we display the `plan.get('@odata.type')` in the stack trace?

Reply:
Thanks